scripts/coastguard-controller.sh

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
remove empty line at start, otherwise #!/bin/bash has no effect.

Reply:
Yupe. missed that.